### DISCUSSION TITLE
Please give presets

### DISCUSSION THREAD

**Discussion by Bibelo** [starship/starship#1538] [Q&A] [answered]
Hello

Please why don't you give examples of configuration?

**Comment by andytom** [comment, marked as answer]
Hi @Bibelo, we do provide example configuration in the [presets section of the starship website](https://starship.rs/presets/), we also have two discussions for starship users to [share their setups](https://github.com/starship/starship/discussions/1107) and [custom modules](https://github.com/starship/starship/discussions/1252).
